Except3/Models/PersonalData.java

```diff
@@ -0,0 +1,72 @@
+package Models;
+import java.lang.Exception;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+//Класс с персональными данными
+public class PersonalData {
+    public final String SURNAME, FAMILYNAME, PATRONYMICNAME, PHONENUMBER, BIRTHDAY, SEX;
+    public final int fieldsNumber = 6;
+    private static boolean isDateValid(String dateString)
+    {   
+        boolean result = false;
+        try
+        {
+            SimpleDateFormat sdf = new SimpleDateFormat(DataStructure.dateFormat);
+            if (sdf.format(sdf.parse(dateString)).equals(dateString)) result = true;
+        }
+        catch (ParseException pe) {}
+        return result;
+    }
+    
+    public PersonalData(String inputString) throws Exception{
+        String surname, familyName, patronymicName, phoneNumber, birthday, sex;
+        surname = "";
+        familyName = ""; 
+        patronymicName = ""; 
+        phoneNumber = ""; 
+        birthday = ""; 
+        sex = "";
+        String[] userData = inputString.split(DataStructure.fieldSeparator);
+        if (userData.length > fieldsNumber) throw new Exception("Число введенных полей с данными больше, чем нужно");
+        if (userData.length < fieldsNumber) throw new Exception("Число введенных полей с данными меньше, чем нужно");
+        for (int index = 0; index < fieldsNumber; index++){
```

Comment:
According the task description order of params is not fixed

Reply:
https://stackoverflow.com/questions/237061/using-regular-expressions-to-extract-a-value-in-java
You can extract values automatically